### ISSUE TITLE
Second Island Problem

### ISSUE THREAD

**Issue by Flippi1990** [BentoBoxWorld/Likes#49]
### Expected behavior

If I create a second island and use visit, 2 are shown but I always go to the first one. 

if i go on second island the 2 visit points change automatic to secon island.

so we need a second visit auto creation when player create second island

### Observed/Actual behavior

on second island creation

### Steps/models to reproduce

create a second island and use command visit

### BentoBox version

v2.3.0

**Comment by tastybento** [comment]
Yes, Likes isn't multi-island aware yet. It needs to be enhanced to support that.
